bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,224 @@
+img[src*="#small"] {
+    width:400px;
+    height:324px;
+}
```

Comment:
Are you sure this will work? The GitHub preview renders this as oridnary text.

Reply:
Right. I resized the image so it does not need a size modifier anymore.